core/util.py

```diff
@@ -159,6 +159,9 @@ def get_apoiase_donors(campain_id):
         finished = len(new) < limit
     return donors
 
+@cached(cache=TTLCache(maxsize=100, ttl=24 * 3600))
+def get_cached_apoiase_donors():
+    return get_apoiase_donors("5ab97be3c3f083c623a26742")
```

Comment:
Acho que vale movermos esse token para o settings como fazemos com a [`SECRET_KEY` por exemplo](https://github.com/turicas/brasil.io/blob/develop/brasilio/settings.py#L18) e também atualizar o [`env.example`](https://github.com/turicas/brasil.io/blob/develop/env.example#L18) com um esse valor padrão. Daí essa linha seria refatorada para:

```suggestion
    return get_apoiase_donors(settings.APOIASE_TOKEN)
```

Reply:
Acho `APOIASE_PROJECT_ID` um melhor nome (não é um token de autenticação).

Reply:
Ok adicionado na settings